waku/waku_test.go

```diff
@@ -1672,13 +1731,19 @@ func TestMailserverCompletionEvent(t *testing.T) {
 				count++
 			case common.EventMailServerRequestCompleted:
 				require.Equal(t, count, len(envelopes),
-					"all envelope.avaiable events mut be recevied before request is compelted")
+					"all envelope.available events mut be received before request is completed")
 				return
 			}
 		}
 	}
 }
 
+func handleError(t *testing.T, err error) {
+	if err != nil {
+		t.Logf("deferred function error: '%s'", err)
+	}
+}
+
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)